ui/components/Filters.js

```diff
@@ -69,6 +81,41 @@ const useStyles = makeStyles(() => ({
 
 }))
 
+/**
+ * Type Definition for View Type
+ * @typedef {"grid" | "table"} TypeView
+ */
+
+/**
+ * ViewSwitch component renders a switch for toggling between
+ * grid and table views
+ * @param {{ view: TypeView, changeView: (view: TypeView) => void }} props
+ */
+function ViewSwitch({ view, changeView }) {
+  console.log(view)
+  return (
+    <ToggleButtonGroup
+      size="small"
+      value={view}
+      exclusive
+      onChange={(_, newView) => changeView(newView)}
+      aria-label="Switch View"
+    >
+      <Tooltip title="Grid view">
+        <ToggleButton value="grid">
+          <GridOnIcon />
+        </ToggleButton>
+      </Tooltip>
+      <Tooltip title="Table view">
+        <ToggleButton value="table">
+          <TableChartIcon />
+        </ToggleButton>
+      </Tooltip>
+    </ToggleButtonGroup>
```

Comment:
This function is exact same is is written in `ui/components/MesheryPatterns.js`. Export that function instead of rewriting